### DISCUSSION TITLE
Is the audit trail already working? if not can we use versioning?

### DISCUSSION THREAD

**Discussion by webmedia1012** [OrchardCMS/OrchardCore#10647] [Q&A]
Hi all, 

Anybody here has implemented versioning or audit trail on published posts? and if yes, how we're you able to achieve it?

Thank you,
Ga

**Comment by john-s-morgan** [comment]
Audit trail is an off-the-shelf function of this product. See https://github.com/OrchardCMS/OrchardCore/issues/4620

[the remark this answers is not in the document]
**Reply by Piedone** [reply]
Thanks Hisham, though I think this is answered :).

[the remark this answers is not in the document]
**Reply by hishamco** [reply]
I didn't see the answer is marked, so that's why I mentioned you

Thanks anyway

[the remark this answers is not in the document]
**Reply by webmedia1012** [reply]
thank you no not answered yet
